example-rules.yml

```diff
@@ -3,12 +3,16 @@ groups:
   rules:
   # The count of CPUs per node, useful for getting CPU time as a percent of total.
   - record: instance:node_cpus:count
-    expr: count(node_cpu{mode="idle"}) without (cpu,mode)
+    expr: count(node_cpu_seconds_total{mode="idle"}) without (cpu,mode)
 
   # CPU in use by CPU.
-  - record: instance_cpu:node_cpu_not_idle:rate5m
-    expr: sum(rate(node_cpu{mode!="idle"}[5m])) without (mode)
+  - record: instance_cpu:node_cpu_seconds_not_idle:rate5m
+    expr: sum(rate(node_cpu_seconds_total{mode!="idle"}[5m])) without (mode)
 
   # CPU in use by mode.
-  - record: instance_mode:node_cpu:rate5m
-    expr: sum(rate(node_cpu[5m])) without (cpu)
+  - record: instance_mode:node_cpu_seconds:rate5m
+    expr: sum(rate(node_cpu_seconds_total[5m])) without (cpu)
+
+  # CPU in use ratio.
+  - record: instance:node_cpu_seconds_total_in_use:ratio
```

Comment:
A ratio would usually be A_per_B. Here it would be node_cpu_seconds_not_idle_per_node_cpu_seconds (as it stands it is not obvious the expression is correct by inspection - and in fact it is wrong as the LHS lacks an instance label in addition to the oddness of dividing a rate5m by a count).

We should probably cheat a bit and do instance:node_cpu_utilisation:ratio

Reply:
`instance:node_cpu_utilization:ratio` sounds fine to me.

Reply:
I fixed the LHS instance label issue.